src/app/components/core/system-alert/system-alert.component.ts

```diff
@@ -249,7 +249,7 @@ export class SystemAlertComponent {
     if (message.includes('restarted')) {
       return "Show Logs";
     }
-    if (message.includes('upgrade')) {
+    if (message.includes('updates') || message.includes('update')) {
```

Comment:
We should first check based on key. 

I understand the problem that service restart does not have a defined key and that still need to rely on text parsing.